packages/messages/src/domains/user.js

```diff
@@ -0,0 +1,14 @@
+const { Schema } = require('mongoose');
+
+const connection = require('../drivers/mongodb/connection');
+
+const User = new Schema({
+  name: { type: String },
+  username: { type: String },
+  avatar: { type: String, required: true },
+  isActive: { type: Boolean, default: true },
+  createdAt: { type: Date, default: Date.now },
```

Comment:
`createdAt` and `updatedAt` are added automatically when using `{ timestamps: true }`, so we can remove them.

Reply:
sureee